### PR TITLE
I made tasks for logical operators

### PR DESCRIPTION
**Pull request by nimdolphin**

### CODE REVIEW

**Comment by kathrynowy** [comment]
logical what?)

File: index.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+// ЗАДАЧИ логические операторы
+
+// Проверка значения из диапазона
+
+if (age >= 14 && age <= 90);
+
+// Проверка значения вне диапазона
+if (!(age >= 14 && age <= 90));
+
+if (age < 14 || age > 90);
```
Comment (kathrynowy) [inline index.js]: why do you need an empty if? add {} and do for example console.log here

we don't write like this never

File: index.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+// ПРОВЕРКА ЛОГИНА
+let login = prompt("Кто там?", "");
+if (login == "Admin") {
```
Comment (kathrynowy) [inline index.js]: we use only strict comparison

File: index.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  console.log("");
+  let password = prompt("пароль?", "");
+  if (password == "я главный") {
+    console.log("Здравствуйте");
+  } else if (password == null) {
```
Comment (kathrynowy) [inline index.js]: we don't check for null

Comment (kathrynowy) [reply index.js]: i've just sent it to TG for you

File: index.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    console.log("отменено");
+  } else {
+    console.log("неверный пароль");
+  }
+} else if (login == null) {
```
Comment (kathrynowy) [inline index.js]: wrong check

please check this if, it won't work

File: index.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+  console.log("неверно");
+}
+// при а = 2 будет верно, при остальных значениях будет неверно.
+
+//  2) Если переменная a равна нулю или равна двум, то прибавьте к ней 7, иначе поделите ее на 10.
+//  Выведите новое значение переменной на экран.
+// Проверьте работу скрипта при a, равном 5, 0, -3, 2.
+
+let f = prompt("число", "");
+if (f == 0 || f == 2) {
```
Comment (kathrynowy) [inline index.js]: remove checking for 0

and use strict comparison

File: index.js [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+  f += 7;
+  console.log(f);
+} else {
+  f /= 10;
+  console.log(f);
+}
+
+let e = 5;
+
+if (e == 0 || e == 2) {
```
Comment (kathrynowy) [inline index.js]: same

File: index.js [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+let b = 0;
+
+if (b == 0 || b == 2) {
+  b += 7;
+  console.log(b);
+} else {
+  b /= 10;
+  console.log(b);
+}
+
```
Comment (kathrynowy) [inline index.js]: you don't need to copy-paste your code.

just change the value of a variable.

File: index.js [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+let d = 2;
+
+if (d == 0 || d == 2) {
+  d += 7;
+  console.log(d);
+} else {
+  d /= 10;
+  console.log(d);
+}
+
```
Comment (kathrynowy) [inline index.js]: remove this copy-paste pls

File: index.js [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+let n = 6;
+
+if (m >= 1 && n >= 3) {
+  m + n;
+  console.log(m + n);
+} else {
+  m - n;
+  console.log(m - n);
+}
+
```
Comment (kathrynowy) [inline index.js]: same here

File: index.js (hunk 3 above)
Comment (kathrynowy) [inline index.js]:
```suggestion
  console.log("");
  
  let password = prompt("пароль?", "");
  
  if (password == "я главный") {
    console.log("Здравствуйте");
  } else if (password == null) {
```

Comment (kathrynowy) [reply index.js]: добавить отступы и удалять за собой все логи

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 2 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 3 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 4 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 5 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 6 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 7 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 8 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 9 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 3 above)
Comment (nimdolphin) [reply index.js]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**